kube-derive/src/custom_resource.rs

```diff
@@ -401,6 +413,9 @@ impl CustomDerive for CustomResource {
                             }
                         })
                     } else {
+                        let gen = schemars::gen::SchemaSettings::openapi3().into_generator();
+                        let schema = gen.into_root_schema_for::<Self>();
```

Comment:
I forgot to test `v1beta1` with this change :man_facepalming:

It's failing to compile because this line is still generated for `v1beta1`. I think the easiest is to generate separate function.